browser-test.html

```diff
@@ -750,9 +750,9 @@
   },
   {
     "name": "version-invalid-must-match-full",
-    "text": "/* ==UserStyle==\n@version 0.1.1.1\n==/UserStyle== */\n",
+    "text": "/* ==UserStyle==\n@version \"0.1.1.1 \"\n==/UserStyle== */\n",
```

Comment:
Do you mean this line? `browser-test.html` is generated by `browser-test.js`. `browser-test.js` collects test cases from `tests/cases` and writes them into a single HTML file. Therefore you can open the webpage with your browser (e.g. Chrome 49), check the console and see whether all tests have passed.

Since data are written with `JSON.stringify`, double quotes get escaped as `\"`.

Reply:
I see, thanks.